server/routes/room-routes.js

```diff
@@ -828,15 +828,13 @@ router.get(
       }),
```

Comment:
@tulir I've made a few additional changes to the tests and surrounding code to make this PR more viable -> https://github.com/matrix-org/matrix-public-archive/pull/274 Sorry for moving ahead of review comments but it was hard to tell exactly what to do otherwise.

Feel free to [cherry-pick those commits](https://github.com/matrix-org/matrix-public-archive/pull/274) (or copy the changes here) so we can merge via this original PR with all of the context.

If you don't have any interest, I can also merge via that PR.

Reply:
Copied the commits here